pkg/frontend/load.go

```diff
@@ -1264,7 +1276,11 @@ func writeBatchToStorage(handler *WriteBatchHandler, force bool) error {
 		//logutil.Infof("----batchBytes %v B %v MB",batchBytes,batchBytes / 1024.0 / 1024.0)
 		//
 		wait_a := time.Now()
+		var unsafePL = (*unsafe.Pointer)(unsafe.Pointer(&handler.startTime))
```

Comment:
1，use atomic.Value
2,  add function into threadInfo for updating time and threadcount.